libs/components/src/popover/popover.component.ts

```diff
@@ -5,11 +5,12 @@ import { Component, EventEmitter, Input, Output, TemplateRef, ViewChild } from "
 
 import { IconButtonModule } from "../icon-button/icon-button.module";
 import { SharedModule } from "../shared/shared.module";
+import { TypographyDirective } from "../typography/typography.directive";
```

Comment:
For consistency, let's change this to 
```suggestion
import { TypographyModule } from "../typography";
```
and on line 13.

(check in the code editor before committing that change because I did it from memory lol)